yarn.lock

```diff
@@ -174,6 +174,14 @@
     "@babel/highlight" "^7.22.13"
     chalk "^2.4.2"
 
+"@babel/code-frame@^7.24.7":
+  version "7.24.7"
+  resolved "https://registry.yarnpkg.com/@babel/code-frame/-/code-frame-7.24.7.tgz#882fd9e09e8ee324e496bd040401c6f046ef4465"
+  integrity sha512-BcYH1CVJBO9tvyIZ2jVeXgSIMvGZ2FDRvDdOIVQyuklNKSsx+eppDEBq/g47Ayw+RqNFE+URvOShmf+f/qwAlA==
+  dependencies:
+    "@babel/highlight" "^7.24.7"
+    picocolors "^1.0.0"
+
```

Comment:
This should have been combined with the block above. Why didn't the resolution do this?

Reply:
they shouldn't be combined, 7.22.13 and 7.24.7 are not the same version
https://github.com/opensearch-project/OpenSearch-Dashboards/blob/9431e115011b99ea8b6c87455871a583b464c081/yarn.lock#L169-L170